activitysim/core/configuration/top.py

```diff
@@ -119,6 +120,11 @@ class OutputTables(PydanticBase):
     h5_store: bool = False
     """Write tables into a single HDF5 store instead of individual CSVs."""
 
+    file_type: str = "csv"
```

Comment:
If we change this line to: `file_type: Literal["csv", "parquet", "h5"] = "csv"` then we don't need the seperate validator function, we get that check for free. and the code will be easier to read.